src/Spa/Generated/Route.elm

```diff
@@ -53,6 +57,7 @@ type Route
     | Music__Videos__Videoid_Int { videoid : Int }
     | Music__Genre__Genre_String { genre : String }
     | Tvshows__Tvshowid_Int { tvshowid : Int }
+    | Tvshows__Seasons__Seasonid_Int { tvshowid : Int, seasonid : Int }
```

Comment:
I'm confused by this being the seasonId, I can't test it, as my setup has too many tvshows. But the old chorus seems to be using the season number.

Reply:
Hi @razzeee,
The routing for the season's page was earlier based on `seasonid`. I have made the modifications and now the routing is based on `season number` like in the old chorus. I have also fixed the naming of `watchedepisode` field.